platform/darwin/src/MGLFeature_Private.h

```diff
@@ -37,3 +37,31 @@ mbgl::Feature mbglFeature(mbgl::Feature feature, id identifier, NSDictionary *at
 NS_DICTIONARY_OF(NSString *, id) *NSDictionaryFeatureForGeometry(NSDictionary *geometry, NSDictionary *attributes, id identifier);
 
 NS_ASSUME_NONNULL_END
+
+#define MGLFeatureInitCoding() \
```

Comment:
Macros should be named in SCREAMING_SNAKE_CASE. 📣🐍 In this case, since this macro is defining a method, it should be named with a verb, like `MGL_DEFINE_FEATURE_INIT_WITH_CODER()`.